### PR TITLE
upgrade xml to kramdown, and add simple Makefile

### PR DESCRIPTION
**Pull request by mcr**

### CODE REVIEW

File: README.md
Comment (furry13) [inline README.md:1]: ANIMA? ;)

File: draft-colitti-ipsecme-esp-ping.mkd
Comment (furry13) [inline draft-colitti-ipsecme-esp-ping.mkd:1]: How did you converted .xml to.mkd? (I'm wondering if I can still edit .xml still and convert to .mkd)

**Review by furry13 — COMMENTED** [review]
I guess README needs to be updated ;)

File: draft-colitti-ipsecme-esp-ping.mkd
Comment (mcr) [reply draft-colitti-ipsecme-esp-ping.mkd:1]: MKD produces XML via kramdown.  You don't edit XML anymore.

File: README.md
Comment (mcr) [reply README.md:1]: Copy and paste, I will fix.

File: draft-colitti-ipsecme-esp-ping.mkd
Comment (furry13) [reply draft-colitti-ipsecme-esp-ping.mkd:1]: I know how to produce xml from mkd, I was wondering if I still can edit .xml and keep it in sync with .mkd (I have slight preference for editing .xml, but can live with markdown as well).

**Comment by mcr** [comment]
furry13 ***@***.***> wrote:
    > I know how to produce xml from mkd, I was wondering if I still can edit
    > .xml and keep it in sync with .mkd (I have slight preference for
    > editing .xml, but can live with markdown as well).

If you want to stick to XML, we can do that. There is no sync.
I find editing XML is like writing assembly. After a couple of 5000+ line
documents in XML, I'd rather not do that again.  This document short enough,
it doesn't really matter.

**Comment by furry13** [comment]
On Fri, Nov 24, 2023 at 12:59 PM Michael Richardson
***@***.***> wrote:
> If you want to stick to XML, we can do that. There is no sync.
> I find editing XML is like writing assembly. After a couple of 5000+ line
> documents in XML, I'd rather not do that again. This document short enough,
> it doesn't really matter.

Don't worry, my preference is not so strong. If you dislike xml, let's
use .md, fine with me, I was just curious :)

-- 
Cheers, Jen Linkova

**Pull request merged**